app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -41,8 +41,16 @@ def button
     end
   end
 
+  def show_list
+    process_show_list
+  end
+
   private
 
+  def display_name
+    ui_lookup(:models => @record.class.base_class.name)
```

Comment:
> "Container Replicators" to "Replicators", according to en.yaml.
> If this is the desired scenario, I also need to change the breadcrumb specs (currently failing for all entities but pods)

imo it's a reasonable change. there's no other replicators in the system (unlike, for example, services), so it doesn't sound like it will cause a confusion.